client/src/partials/section_bilan/section_bilan.html

```diff
@@ -1,127 +1,195 @@
-<header data-header>
-  {{ "SECTION_BILAN.TITLE" | translate }}
-</header>
-
-<nav>
-  <div class="pull-left">
-    <ol class="breadcrumb">
-      <li><a href="#/"><span class="glyphicon glyphicon-home"></span></a></li>
+<div class="flex-header">
+  <div class="bhima-title">
+    <ol class="headercrumb">
+      <li class="static">{{ "TREE.ACCOUNTING" | translate }}</li>
       <li class="active">{{ "SECTION_BILAN.TITLE" | translate }}</li>
     </ol>
   </div>
-
-  <div class="pull-right">
-    <button class="btn btn-sm btn-default" ng-click="new()">
-      <span class="glyphicon glyphicon-plus-sign"></span>
-      {{ "SECTION_BILAN.ADD" | translate }}
-    </button>
-  </div>
-</nav>
-
-<main>
-  <div class="row">
-
-    <div class="col-xs-6">
-      <div class="panel panel-primary">
-        <div class="panel-heading">
-          <span class="glyphicon glyphicon-home"></span> {{ "SECTION_BILAN.ALL" | translate }}
+</div>
+
+<!-- Links to the Section Bilan Chart as well as the button for adding an sectionBilan -->
+<div class="flex-util">
+  <button ng-click="sectionBilanCtrl.create()" class="btn btn-default btn-sm" data-method="create">
+    <span class="glyphicon glyphicon-plus-sign"></span> {{ "SECTION_BILAN.ADD" | translate }}
+  </button>
+</div>
+
+
+<div class="flex-content">
+  <div class="container-fluid">
+    <div class="row">
+      <!-- Complete list of registred Section Bilan -->
+      <div class="col-xs-7">
+        <div class="panel panel-default" style="overflow: auto; max-height: 500px;">
+          <div class="panel-heading">
+            {{ "SECTION_BILAN.ALL" | translate }}
+          </div>
+
+          <table class="table table-condensed">
+            <thead>
+              <tr>
+                <th>{{ "COLUMNS.NR" | translate }}</th>
+                <th>{{ "COLUMNS.TEXT" | translate }}</th>
+                <th>{{ "COLUMNS.POSITION" | translate }}</th>
+                <th></th>
+                <th colspan="2">{{ "COLUMNS.ACTION" | translate }}</th>
+              </tr>
+            </thead>
+            <tbody>
+              <tr ng-if="sectionBilanCtrl.session.loading" class="text-center">
+                <td colspan="6"><loading-indicator></loading-indicator></td>
+              </tr>
+              <tr 
+              ng-repeat="sectionBilan in sectionBilanCtrl.sectionBilans | orderBy:'number' | orderBy:'text' " 
+              ng-class="{'rowSelected' : sectionBilanCtrl.sectionBilan.id === sectionBilan.id}" >
+                <td>{{ $index+1 }}</td>
+                <td>{{ sectionBilan.text }}</td>
+                <td>{{sectionBilan.position}}</td>
+                <td>
+                  <b>
+                    {{ (sectionBilan.is_actif == 1) ? sectionBilanCtrl.doTranslate('SECTION_BILAN.ASSET') : sectionBilanCtrl.doTranslate('SECTION_BILAN.PASSIVE') }}
```

Comment:
This would be much easier to understand if this was two lines of html with `ng-show`

For example: 
```js
<b ng-show="sectionBilan.is_actif">...
<b ng-hide="sectionBilan.is_actif">...
```

The ternary conditional (?) is quite difficult to read in templates.